client/packages/invoices/src/InboundShipment/DetailView/modals/InboundServiceLineEdit/useDraftServiceLines.ts

```diff
@@ -1,17 +1,16 @@
 import React, { useEffect } from 'react';
 import { RecordPatch } from '@openmsupply-client/common';
-import {
-  toItemWithPackSize,
-  useDefaultServiceItem,
-} from '@openmsupply-client/system';
+import { toItemWithPackSize, useItem } from '@openmsupply-client/system';
 import { useInbound } from '../../../api';
 import { DraftInboundLine } from './../../../../types';
 import { CreateDraft } from '../utils';
 
 export const useDraftServiceLines = () => {
   const { id } = useInbound.document.fields('id');
   const { data: lines } = useInbound.lines.serviceLines();
-  const { defaultServiceItem, isLoading } = useDefaultServiceItem();
+  const {
+    serviceItem: { data: defaultServiceItem, isLoading },
+  } = useItem();
```

Comment:
Something is going awry - on `develop` I cannot add any service charges, says there are non available for the store. On this branch the button is enabled, I can add an item to the list, but get an error on pressing save:

<img width="896" height="501" alt="Image" src="https://github.com/user-attachments/assets/a18d3fdf-be8b-46fe-a047-f6821604da5a" />

<img width="374" height="86" alt="Image" src="https://github.com/user-attachments/assets/4a1d132c-604a-4a03-8197-7814eeedbb61" />

Reply:
Fixed~

<img width="884" height="280" alt="Screenshot 2025-08-12 at 5 00 34 PM" src="https://github.com/user-attachments/assets/c67090d5-0388-46c6-9646-15de2288ed8b" />